packages/cli/src/commands/test.ts

```
@@ -47,12 +58,28 @@ export const testCommand = new Command('test')
+      // Parse manual scenarios from markdown
+      if (scenarios.length === 0) {
+        const scenarioMatches = specContent.matchAll(
+          /###\s*Scenario\s*\d*:?\s*(.+?)\n[\s\S]*?\*\*Given\*\*\s*(.+?)\n[\s\S]*?\*\*When\*\*\s*(.+?)\n[\s\S]*?\*\*Then\*\*\s*(.+?)(?=\n###|$)/gi
+        );
+        
+        for (const match of scenarioMatches) {
+          scenarios.push({
+            id: `SC-${scenarios.length + 1}`,
+            given: match[2].trim(),
+            when: match[3].trim(),
+            then: match[4].trim()
```

Biome (2.3.14)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)